src/containers/TransferOwnership/TransferOwnership.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useState } from 'react';
+
+//- Style Imports
+import styles from './TransferOwnership.module.css';
+
+//- Component Imports
+import { Overlay, EtherInput, FutureButton, Image } from 'components';
+
+//- Library Imports
+/*
+import { getMetadata } from 'lib/metadata';
+*/
+import { randomImage, randomName } from 'lib/Random';
+import { useTransferProvider } from 'lib/providers/TransferProvider';
+import { Member } from '../../components';
+
+type TransferOwnershipProps = {
+	name: string;
+	image: string;
+	domain: string;
+	creatorId: string;
+	onModalChange: (arg: boolean) => void;
+};
+
+const TransferOwnership: React.FC<TransferOwnershipProps> = ({
+	image,
+	name,
+	domain,
+	creatorId,
+	onModalChange,
+}) => {
+	// State
+	const [walletAddress, setWalletAddress] = useState('');
+	const [hasAccepted, setHasAccepted] = useState(false); // Toggle confirmation overlay
+
+	// Provider
+	const { submit } = useTransferProvider();
+
+	// Form validation
+	const isEthAddress = (text: string) =>
+		/^0x[a-fA-F0-9]{40}$/.test(String(text).toLowerCase());
+	const valid = isEthAddress(walletAddress);
+
+	const accept = () => setHasAccepted(true);
+	const cancel = () => setHasAccepted(false);
+
+	const submitTransfer = () => {
+		onModalChange(false);
+
+		submit({
+			name,
+			image,
+			domain,
+			creatorId,
+			walletAddress
+		});
+	};
+
+	const confirm = () => submitTransfer();
+
+	// Todo
```

Comment:
Not sure what this todo is supposed to be doing, and if it's related to the commented out code

Reply:
I've removed this commented out code as it's no longer needed